tests/sentry/api/endpoints/test_sentry_app_avatar.py

```diff
@@ -0,0 +1,47 @@
+from base64 import b64encode
+
+from sentry.models import SentryAppAvatar
+from sentry.testutils import APITestCase
+
+
+class SentryAppAvatarTestBase(APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.unpublished_app = self.create_sentry_app(name="Meow", organization=self.organization)
+        self.endpoint = "sentry-api-0-sentry-app-avatar"
+
+        self.login_as(self.user)
+
+
+class SentryAppAvatarTest(SentryAppAvatarTestBase):
+    def test_get(self):
+        with self.feature("organizations:sentry-app-logo-upload"):
+            response = self.get_success_response(self.unpublished_app.slug)
+            assert response.data["uuid"] == str(self.unpublished_app.uuid)
+
+
+class SentryAppAvatarPutTest(SentryAppAvatarTestBase):
+    method = "put"
+
+    def test_upload(self):
+        with self.feature("organizations:sentry-app-logo-upload"):
+            data = {
+                "color": 1,
+                "avatar_type": "upload",
+                "avatar_photo": b64encode(self.load_fixture("avatar.svg")),
+            }
+            resp = self.get_success_response(self.unpublished_app.slug, **data)
+
+        avatar = SentryAppAvatar.objects.get(sentry_app=self.unpublished_app)
+        assert avatar.file_id
+        assert avatar.get_avatar_type_display() == "upload"
+        assert resp.data["avatar"]["avatarType"] == 1
+        assert resp.data["avatar"]["avatarUuid"] is not None
+        assert resp.data["avatar"]["color"] is True
+
+    def test_put_bad(self):
+        SentryAppAvatar.objects.create(sentry_app=self.unpublished_app)
+        with self.feature("organizations:sentry-app-logo-upload"):
+            self.get_error_response(
+                self.unpublished_app.slug, avatar_type="upload", status_code=400
+            )
```

Comment:
We should probably add tests for rejecting based on the SVG attributes as well. I mentioned above some concerns with viewBox, but we can still test things like `<script>`, `onload=`, `width=` and `height=`